install/kubernetes/cilium/templates/spire/agent/daemonset.yaml

```diff
@@ -86,6 +86,8 @@ spec:
       {{- with .Values.authentication.mutual.spire.install.agent.tolerations }}
       tolerations:
         {{- toYaml . | trim | nindent 8 }}
+        - key: {{ .Values.agentNotReadyTaintKey }}
```

Comment:
this won't work within the `with` block

Something like `$.Values.agentNotReadyTaintKey` should do it

Reply:
good catch!